README.md

```diff
@@ -9,12 +9,66 @@ It deals with correcting the prosody and intonation.
 pip install git+https://github.com/sign-language-processing/fluent-pose-synthesis
 ```
 
-To make an unfluent pose sequence into a fluent pose sequence:
+
+### 1. Environment Setup
+
+We recommend using `conda` for managing dependencies.
+
+```bash
+# Clone the repository
+pip install git+https://github.com/sign-language-processing/fluent-pose-synthesis
+cd fluent-pose-synthesis
+
+# Create and activate environment
+conda env create -f environment.yml
+conda activate fluent-pose
+```
```

Comment:
then there should be a `pip install .`

Ideally, there shouldn't be `environment.yml` - the necessary dependencies should be speicificed in the `pyproject.toml`